edx_sysadmin/utils/markup.py

```diff
@@ -0,0 +1,34 @@
+"""
+Utilities for use in Mako markup.
+"""
+
+
+import markupsafe
+
+# Text() can be used to declare a string as plain text, as HTML() is used
+# for HTML.  It simply wraps markupsafe's escape, which will HTML-escape if
+# it isn't already escaped.
+Text = markupsafe.escape                        # pylint: disable=invalid-name
+
+
+def HTML(html):                                 # pylint: disable=invalid-name
```

Comment:
Just curious: do we need this even if we are not using `mako` templates?

Reply:
markupsafe is used for markuping the HTML content. do you think it is not required ?

Reply:
Doesn't Jinja/django templates do that themselves?

Reply:
not harm to use markupsafe markup over HTML.